src/Application/ApplicationFileProcessor.php

```diff
@@ -113,4 +127,34 @@ private function printFile(File $file): void
         $this->smartFileSystem->dumpFile($smartFileInfo->getPathname(), $file->getFileContent());
         $this->smartFileSystem->chmod($smartFileInfo->getRealPath(), $smartFileInfo->getPerms());
     }
+
+    /**
+     * Inspired by @see https://github.com/phpstan/phpstan-src/blob/89af4e7db257750cdee5d4259ad312941b6b25e8/src/Analyser/Analyser.php#L134
+     */
+    private function configureCustomErrorHandler(): void
+    {
+        $errorHandlerCallback = function (int $code, string $message, string $file, int $line): bool {
+            $reportingLevel = error_reporting();
+            if ($reportingLevel === 0) {
```

Comment:
you should can use `(error_reporting() & $errno) === 0` or `(! (error_reporting() & $errno))`, see

https://github.com/samsonasik/ErrorHeroModule/blob/18b691e880d6e068dd94d17bffcfdd54518d2a4a/src/HeroTrait.php#L112

I verify error handling a lot for this case.